libs/features/personalization/personalization.js

```diff
@@ -156,7 +161,7 @@ const COMMANDS = {
 };
 
 function checkSelectorType(selector) {
-  return selector?.includes('/fragments/') ? 'fragment' : 'css';
+  return selector?.startsWith('/') || selector?.startsWith('http') ? 'fragment' : 'css';
```

Comment:
If it's authored as a path without `/fragments/` it would fail right?  I wonder if we should have a validator in the mep manifest that requires `/fragments/`?

Reply:
the issue is that gnav has links to menus that are a lot like fragments, but are not necessarily in a fragment folder.  So I was trying to loosen the test.  But maybe we could instead check for the presence of a folder name that is always in gnav?  But I figured that if we use this for other blocks in the future, I'm mostly trying to check if it's a URL.